src/panels/config/automation/action/ha-automation-action-row.ts

```diff
@@ -441,32 +461,54 @@ export default class HaAutomationActionRow extends LitElement {
     this._yamlMode = !this._yamlMode;
   }
 
+  public get expanded() {
+    return this._expanded;
+  }
+
+  public toggleExpanded() {
+    this._expanded = !this._expanded;
+  }
+
   static get styles(): CSSResultGroup {
     return [
       haStyle,
       css`
+        .card-summary {
+          display: flex;
+          align-items: center;
+        }
+        .card-summary .name {
+          flex: 1;
+          cursor: pointer;
+        }
+        .card-summary ha-button-menu {
+          --mdc-theme-text-primary-on-background: var(--primary-text-color);
+        }
+
         .disabled {
           opacity: 0.5;
           pointer-events: none;
         }
         .card-content {
           padding-top: 16px;
           margin-top: 0;
+          display: none;
```

Comment:
I used display: none instead of removing the DOM elements so the state of the DOM elements remains stored. Although I think there was a decorator for this too.

Reply:
https://lit.dev/docs/templates/directives/#caching-and-change-detection